From: Steve French <stfrench@microsoft.com>
Date: Wed, 1 Aug 2018 22:34:04 -0500
Subject: [PATCH] smb3: fix minor debug output for CONFIG_CIFS_STATS
Git-commit: d258650004617fbd5dfe97d77d26fb37187d06e9
Patch-mainline: v4.18
References: bsc#1144333

CONFIG_CIFS_STATS is now always enabled (to simplify the
code and since the STATS are important for some common
customer use cases and also debugging), but needed one
minor change so that STATS shows as enabled in the debug
output in /proc/fs/cifs/DebugData, otherwise it could
get confusing with STATS no longer showing up in the
"Features" list in /proc/fs/cifs/DebugData when basic
stats were in fact available.

Signed-off-by: Steve French <stfrench@microsoft.com>
Reviewed-by: Ronnie Sahlberg <lsahlber@redhat.com>
Acked-by: Aurelien Aptel <aaptel@suse.com>
---
 fs/cifs/cifs_debug.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fs/cifs/cifs_debug.c b/fs/cifs/cifs_debug.c
index e074820bd4ed..748cabd6d20b 100644
--- a/fs/cifs/cifs_debug.c
+++ b/fs/cifs/cifs_debug.c
@@ -170,7 +170,7 @@ static int cifs_debug_data_proc_show(struct seq_file *m, void *v)
 #endif
 #ifdef CONFIG_CIFS_STATS2
 	seq_printf(m, ",STATS2");
-#elif defined(CONFIG_CIFS_STATS)
+#else
 	seq_printf(m, ",STATS");
 #endif
 #ifdef CONFIG_CIFS_DEBUG2
-- 
2.16.4

